### ISSUE TITLE
/voxblox_skeletonizer/sparse_graph

### ISSUE THREAD

**Issue by JinRhee** [snt-arg/lidar_situational_graphs#7]
Hello,
Thank you for sharing great work!

I am trying to run your work on my local machine using the "old" room segmentation method (i.e. making use of the `/room_segmentation_node` rather than `situational_graphs_reasoning`). I am able to launch `s_graphs_launch.py` in ROS2 Foxy with the argument `room_segmentation:=old` and can see that the correct node is active. However, room segmentation does not appear to be working and I see that the topic `/voxblox_skeletonizer/sparse_graph` has no publishers (while `/room_segmentation_node` subscribes to it.) I understand that this free space graph is used within the room segmentation process.

I also have a ROS1 workspace separately built, but am having trouble understanding which node (whether in the ROS2 or ROS1 workspace) is supposed to publish the above topic. Though I see that there are voxblox and skeletonizer packages within `s_graphs_utils` (from .rosinstall_ros1), none of the launch files included within `src/launch` seem to reference any voxblox/voxblox skeletonizer packages.

How would I be able to get the skeletonized graph topic/data? Should I be running packages separately from situational graphs?

Any help or pointers would be much appreciated!

**Comment by hridaybavle** [comment]
Hello, 

You are right that in the newer version we removed the instructions to launch the voxblox skeleton node. Going back to an old commit I found the [mprocs](https://github.com/snt-arg/lidar_situational_graphs/blob/5ac73ddc058876b736b1ee055cf395785ee0e064/.real_mprocs.yaml#L10) file where we were  launching the voxblox skeleton node. Try launching it in a separate terminal and see if that works.
